code/style.css

```diff
@@ -1,7 +1,201 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+body {
+  margin: 0;
+}
+
+.hero-section {
+  position: relative;
+  font-family: fantasy;
+  color: white;
+  margin: 0;
+  background: rgba(106, 142, 117, 0.7);
+}
+
+.hero-image {
+  width: 100vw;
```

Comment:
do not use `vw` because the scroll bar makes your image wider than the visible part of the page, so it stretches the whole page on the right side.

Reply:
I used `Pesticide` Chrome plugin and you can see that your hero is narrower than the image.
![image](https://user-images.githubusercontent.com/50926159/122640020-5cf81e80-d0fd-11eb-8452-bdac9ea5b275.png)

Reply:
Hi Karol, thanks! I removed view width from the hero image (and the user section) and replaced it with %. I also downloaded the pesticide plugin. I've tried to recreate the white margin on the right side, using both vw and % but I still can't see it. I have also looked at the page from a pc laptop and can't recreate the margin. Best, Malin